### ISSUE TITLE
The Meaning of Loss in each tutorial.

### ISSUE THREAD

**Issue by liuzhi136** [zackchase/mxnet-the-straight-dope#221]
I noted the way you compute loss as below in each tutorial that you write the script from scratch. Can you explain why are you use this equation to accumulate loss?@zackchase
moving_loss = .99 * moving_loss + .01 * nd.mean(loss).asscalar()
